core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -256,7 +256,9 @@ require('services/PlaythroughIssuesService.ts');
 require('services/SiteAnalyticsService.ts');
 require('services/StateTopAnswersStatsBackendApiService.ts');
 require('services/StateTopAnswersStatsService.ts');
-
+require(
+  'pages/exploration-editor-page/editor-navigation/' +
+  'editor-navigation-dropdown.directive.ts');
```

Comment:
Move this after the next require, we are sorting requires alphabetically.

Reply:
I have moved this require to the next line.